README.md

```diff
@@ -18,7 +18,8 @@ try {
   require('@dynatrace/oneagent')({
     environmentid: '<environmentid>',
     apitoken: '<paastoken>',
-    endpoint: '<endpoint url>' // specify endpoint url - not needed for SaaS customers
+    endpoint: '<endpoint url>', // specify endpoint url - not needed for SaaS customers
+    options: { rejectUnathorized: true } // for self-signed certificate use `rejectUnathorized: false`
```

Comment:
I think this requires more docs to avoid confusions.
We should clearly state that this setting effects only the single request to `/v1/deployment/installer/agent/connectioninfo` and nothing else.
We should also mention the default if not used.